Vagrantfile

```diff
@@ -9,13 +9,19 @@ main_k3s_ip_address = "192.168.56.2"
 main_k3s_port = 6443
 fqdn = "k3ces.local"
 docker_registry_namespace = "ecosystem"
```

Comment:
Maybe docker_registry_namespace could be renamed to `ces_namespace` because it is no just used for the docker-registry but also to install the CES